Gemfile

```diff
@@ -14,6 +15,7 @@ gem 'devise'
 
 
 group :development, :test do
+  gem 'dotenv-rails'
```

Comment:
You dont need this gem if you are using rails credentials.

Reply:
Well this is what made it work for us!

Reply:
They are NOT using encrypted credentials @magnus-thor